### PR TITLE
[Merged by Bors] - feat: the shift on the opposite category

### PR DESCRIPTION
**Pull request by joelriou**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Triangulated/Opposite.lean [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+private abbrev OppositeShiftAux :=
+  PullbackShift (OppositeShift C ℤ)
+    (AddMonoidHom.mk' (fun (n : ℤ) => -n) (by intros; dsimp; linarith))
+
+/-- The category `Cᵒᵖ` is equipped with the shift such that the shift by `n` on `Cᵒᵖ`
+corresponds to the shift by `-n` on `C`. -/
+noncomputable scoped instance : HasShift Cᵒᵖ ℤ :=
+  (inferInstance : HasShift (OppositeShiftAux C) ℤ)
+
+instance [Preadditive C] [∀ (n : ℤ), (shiftFunctor C n).Additive] (n : ℤ) :
```
Comment (jcommelin) [inline Mathlib/CategoryTheory/Triangulated/Opposite.lean:59]: Should this also be a scoped instance?

Comment (joelriou) [reply Mathlib/CategoryTheory/Triangulated/Opposite.lean:59]: I had tried, but the linter complained...

Comment (jcommelin) [reply Mathlib/CategoryTheory/Triangulated/Opposite.lean:59]: In that case, let's kick this on the queue.

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Pull request closed without merging**